agent/acl_test.go

```diff
@@ -274,79 +274,82 @@ func TestACL_Down_Allow(t *testing.T) {
 
 func TestACL_Down_Extend(t *testing.T) {
 	t.Parallel()
-	a := NewTestAgent(t.Name(), TestACLConfig()+`
-		acl_down_policy = "extend-cache"
+	aclExtendPolicies := []string{"extend-cache", "async-cache"}
+	for _, aclDownPolicy := range aclExtendPolicies {
+		a := NewTestAgent(t.Name(), TestACLConfig()+`
+		acl_down_policy = "`+aclDownPolicy+`"
 		acl_enforce_version_8 = true
 	`)
```

Comment:
OPT: the identation is really confusing here as it is quite easy to confuse ")" and "}"

Reply:
@yfouquet Fixed in next patch